lib/pry/config.rb

```diff
@@ -31,5 +31,14 @@ def call
     def self.shortcuts
       Convenience::SHORTCUTS
     end
+
+    def prompt=(obj)
+      is_prompt_object = lambda do |prompt|
```

Comment:
@kyrylo 
if prompts were a Struct or similar, we could use `is_a?` here instead.
eg: `Pry::Prompt::PromptInfo === obj`

Reply:
this diff is outdated